lib/graphql/invalid_null_error.rb

```diff
@@ -24,9 +24,9 @@ def to_h
       { "message" => message }
     end
 
-    # @return [Boolean] Whether the null in question was caused by another error
+    # @deprecated always false
     def parent_error?
```

Comment:
The only time this was every `true` in the past was inside the execution strategy.  I kept this because graphql-batch previously needed to [handle this InvalidNullError](https://github.com/Shopify/graphql-batch/blob/v0.3.0/lib/graphql/batch/execution_strategy.rb#L7) in case it was raised from a promise.